composeApp/src/commonMain/kotlin/org/ooni/probe/ui/dashboard/DashboardViewModel.kt

```diff
@@ -73,7 +77,19 @@ class DashboardViewModel(
             TaskSettings(
                 name = "web_connectivity",
                 inputs = listOf("https://ooni.org"),
-                logLevel = "DEBUG2",
+                logLevel = TaskLogLevel.Info,
+                options =
+                    TaskSettings.Options(
+                        noCollector = true,
+                        softwareName = Config.BASE_SOFTWARE_NAME,
```

Comment:
This `software_name` is set to `ooniprobe-android-unattended` when the tests are run from the background and `ooniprobe-android`.

See *Raw Measurement Data* in https://explorer.ooni.org/m/20240731235959.972250_IE_webconnectivity_50b43329c29ed566

Reply:
For now, you can disregard the `DashboardViewModel.kt` code. I just kept the "Run Test" working to see if everything was fine. But we're never collecting/uploading, we still need to do the Engine SDK to properly set all configs.